VoodooTrackpoint/VoodooTrackpointMessages.h

```diff
@@ -12,6 +12,8 @@
 #define kIOMessageVoodooTrackpointRelativePointer iokit_vendor_specific_msg(430)
 #define kIOMessageVoodooTrackpointScrollWheel iokit_vendor_specific_msg(431)
 
+#define VOODOO_TRACKPOINT_IDENTIFIER "VoodooTrackpointIdentifier"
```

Comment:
What is this used for?

Reply:
When attaching, it's used to identify that it's VoodooTrackpoint and not something else. It's a property that gets set in VoodooTrackpoint that other kexts look for. It allows us to just include the trackpoint message header file and not care about the Trackpoint device class at all - I think it'd make it safer against updates to VoodooTrackpoint as well since the other kexts don't really need to care about binary changes.